packages/bpk-component-rating/src/BpkRatingSizes.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+const RATING_SIZES = { base: 'base', lg: 'lg', sm: 'sm' };
```

Comment:
I know we spoke about this yesterday but we shouldn't be including this if it doesn't do anything yet. 

Imagine if we released this right now - if a consumer looks at the source code they'll see that they can pass any 'base', 'lg' or 'sm' into the component, but nothing will happen if they do provide one of these values, which is a poor experience for them and violates the principle of least surprise.

My advice would be to:
- Not export `RATING_SIZES` from the main package until it's useful for consumers
- Comment out the prop type and flow type from the code for now with a quick `// TODO make use of this` comment next to it

That way we don't lose the code you've written but we don't confuse consumers either

Reply:
That is true but in this case as it will be coming really soon we keep this as is? The README.md does not mention this prop type so unless someone digs into the core code this won't be known 😄

Reply:
If their IDE suggest props based on Flow types they'd still see it..

Reply:
I agree with @shaundon that we shouldn't release code that isn't functional - however we won't be releasing this because of `"private": true` in the `package.json`.

As a general rule though - I also think including non-functional code bloats PR's - so in the interest of smaller PR's I'm also not a fan

Reply:
Sure thing! Will kill it out for now and then when I do the ticket to add other sizes after this PR is merged will just copt and paste it back in

Reply:
Commented out the code so it exists but has TODOs around it 😄